Tests/FluentTests/Utilities/Atom.swift

```diff
@@ -17,7 +17,7 @@ struct Atom: Entity {
         groupId = try node.extract("group_id")
     }
 
-    func makeNode() throws -> Node {
+    func makeNode(context: Context = EmptyNode) throws -> Node {
```

Comment:
There's already a default on protocol extension that does this. The default arg is really only useful on implementations that don't throw. For example, you can do `true.makeNode()` w/o throws, but the protocol extension has to throw